e2e-tests/README.md

```diff
@@ -1,5 +1,11 @@
-## Prerequirements
-CentOS
+# Building and testing the Operator
```

Comment:
I think we need to mention that user should have EKS/GKE/OpenShift or minikube (and versions which we officially support and link to instruction on how to deploy  e.g. https://www.percona.com/doc/kubernetes-operator-for-pxc/eks.html ) cluster to have the possibility to run operator.